xml/operations-maintenance-controller-pit_database_recovery.xml

```diff
@@ -283,4 +283,60 @@ hostname = {{ hostname of the first &mariadb; node }}</screen>
    </step>
   </procedure>
  </section>
+ <section>
+  <title>Point-in-Time Cassandra Recovery</title>
+  <para>
+   A node may have been removed either due to an intentional action in the
+   &lcm; Admin UI or as a result of a fatal hardware event that requires a
+   server to be replaced. In either case, the entry for the failed or deleted
+   node should be removed from Cassandra before a new node is brought up.
+  </para>
+  <para>
+   The following steps should be taken before enabling and deploying the
+   replacement node.
+  </para>
+  <procedure>
+   <step>
+    <para>
+     Determine the IP address of the node that was removed or is being replaced.
+    </para>
+   </step>
+   <step>
+    <para>
+     On one of the functional &cassandra; control plane nodes, log in as the
+     <literal>ardana</literal> user.
+    </para>
+   </step>
+   <step>
+    <para>
+     Run the command <command>nodetool status</command> to display a list of
+     &cassandra; nodes.
+    </para>
+   </step>
+   <step>
+    <para>
+     If the node that has been removed (no IP address matches that of the
+     removed node) is not in the list, skip the next step.
+    </para>
+   </step>
+   <step>
+    <para>
+     If the node that was removed is still in the list, copy its node
+     <replaceable>ID</replaceable>.
+    </para>
+   </step>
+   <step>
+    <para>
+     Run the command <command>nodetool removenode
+     <replaceable>ID</replaceable></command>'
+    </para>
+   </step>
+  </procedure>
+  <para>
+   After any obsolete node entries have been removed, the replacement node can
+   be deployed as usual (for more information, see <xref
+   linkend="cont_planned"/>. The new &cassandra; node will be able to join the
```

Comment:
Looks like it might be missing a ) here.

Reply:
Got a link. Can we also add http://cassandra.apache.org/doc/latest/operating/topo_changes.html ?